jsk_recognition_msgs/msg/PeoplePose.msg

```diff
@@ -0,0 +1,2 @@
+string[] limb_names
+geometry_msgs/Point32[] points
```

Comment:
Why not we use `geometry_msgs/Pose` with nan rotation values ((0, 0, 0, 0) for quaternion),
expecting the future work of estimating *rotation* of human limbs.